sdk/communication/Azure.Communication.Identity/src/CommunicationIdentityClient.cs

```diff
@@ -62,28 +65,37 @@ public CommunicationIdentityClient(Uri endpoint, TokenCredential tokenCredential
         private CommunicationIdentityClient(CommunicationIdentityClientOptions options, ConnectionString connectionString)
         {
             _clientDiagnostics = new ClientDiagnostics(options);
+            _pipeline = options.BuildHttpPipeline(connectionString);
+            _endpoint = connectionString.GetRequired("endpoint");
             RestClient = new CommunicationIdentityRestClient(
                 _clientDiagnostics,
-                options.BuildHttpPipeline(connectionString),
-                connectionString.GetRequired("endpoint"));
+                _pipeline,
+                _endpoint,
+                apiVersion: "2021-03-07");
         }
 
         private CommunicationIdentityClient(Uri endpoint, CommunicationIdentityClientOptions options, AzureKeyCredential credential)
```

Comment:
Your baseline code is out of date, you should probably rebase onto latest master

Reply:
Yes, I think there were some changes overnight, which I didn't get a chance to merge. I will send a new iteration with merged changes.

Reply:
There are some old changes as well around how the client gets constructed

Reply:
Merged with latest changes.